src/player.cpp

```diff
@@ -10239,11 +10239,18 @@ bool player::wield(item* it, bool autodrop)
         return false;
     }
 
-    if( it->is_two_handed(*this) && !has_two_arms() ) {
-        add_msg(m_info, _("You cannot wield a %s with only one arm."),
+    if (!has_two_arms()) {
+        if (it->has_flag("ALWAYS_TWOHAND")) {
+        add_msg(m_info, _("You cannot wield a %s with only one arm, as it is handled only two-handed."),
```

Comment:
Similar here. Should be simpler, like
"The %s can't be wielded with only one arm"